src/Parser/MistweaverMonk/Modules/Items/Velens.js

```diff
@@ -34,7 +34,25 @@ class Velens extends Module {
       return;
     }
 
-    this.healing += calculateEffectiveHealing(event, LEGENDARY_VELENS_HEALING_INCREASE);
+    // Rewriting the effectiveHealing function here, to remove the final step of removing the raw overheal from the effective healing of the trinket.
```

Comment:
So the reason I did this was because the way the calculation looks to process is that is collects all healing amount, absorbed amount, and overheal amount.  It then calculates the Velen's 15% contribution on this full amount.  After that it removes the raw overheal from the contribution.

I think removing the raw overhealing from the effective healing before calculating the Velen's impact would be a cleaner solution.  That is the approach I took.  Just because raw healing is increased by 15%, the overhealing on that has no bearing on the effectiveness so removing it from the equation entirely just seems to make more sense.

I also think the same applies to Drapes contribution to your healing.  The raw overheal amount shouldn't be in the calculation to start.